ray-operator/main.go

```diff
@@ -56,7 +61,10 @@ func main() {
 		"Temporary feature flag - to be deleted after testing")
 	flag.BoolVar(&ray.ForcedClusterUpgrade, "forced-cluster-upgrade", false,
 		"Forced cluster upgrade flag")
-	opts := zap.Options{
+	flag.StringVar(&logFile, "log_file", "",
```

Comment:
let's make the param pattern consistent.  `log_file` -> `log-file-path`.   let's use hyphen instead of underline

Reply:
fixed